src/guides/v2.3/inventory/inventory-cli-reference.md

```diff
@@ -109,30 +110,44 @@ Following reservations were created:
 - Product bike-123 was compensated by +2.000000 for stock 1
 ```
 
-You can run both commands by piping `list-inconsistencies` and `create-compensations` to detect inconsistencies and immediately create compensations. Use the `-r` command option to generate and submit the raw data to `create-compensations`.
+### Detect inconsistencies and create compensations
+
+You can detect inconsistences and immediately create compensations by using a pipe to run both the `list-inconsistencies` and `create-compensations`. Use the `-r` command option to generate and submit the raw data to `create-compensations`.
 
 ```bash
 bin/magento inventory:reservation:list-inconsistencies -r | bin/magento inventory:reservation:create-compensations
 ```
 
 Example response:
 
-```terminal
+```bash
 bin/magento inventory:reservation:list-inconsistencies -r | bin/magento inventory:reservation:create-compensations
+```
 
+```terminal
 Following reservations were created:
 - Product bike-123 was compensated by +2.000000 for stock 1
 - Product bikehat-456 was compensated by +1.000000 for stock 1
 ```
 
 After updates complete, run the list command to verify:
 
-```terminal
+```bash
 bin/magento inventory:reservation:list-inconsistencies -r
 
 No order inconsistencies were found.
 ```
 
+You can also pipe the commands to detect inconsistencies and create compensations for only incomplete (`-i`) or complete (`-c`) orders.
```

Comment:
The line "No order inconsistencies were found." above is output. Separate the output  from the input, as you did for the previous example.